InvenTree/InvenTree/tasks.py

```diff
@@ -563,6 +563,16 @@ def send_email(subject, body, recipients, from_email=None, html_message=None):
     if type(recipients) == str:
         recipients = [recipients]
 
+    import InvenTree.ready
+
+    if InvenTree.ready.isImportingData():
+        # If we are importing data, don't send emails
+        return
+
+    if InvenTree.ready.isInTestMode():
+        # If we are running in test mode, don't send emails
+        return
```

Comment:
Question: Should we perhaps test if email delivery is working somewhere?

Reply:
I'm going to revert this particular line anyway, it's breaking CI :)